engine/src/exec-pasteboard.cpp

```diff
@@ -279,7 +279,7 @@ MCTransferType MCPasteboardFindTransferTypeForLegacyClipboard(const MCClipboard*
     // passed that the behaviour can now be corrected but, until then, it is
     // disabled and the "text over images" method is used instead.
     //
-    if (false)
+    if (/* DISABLES CODE */ (false))
```

Comment:
It's a good idea and I'm glad that Xcode suggested it, so I think it's worth making official 😉